ui/src/components/PathList/PathListItem.tsx

```diff
@@ -0,0 +1,129 @@
+import { Accordion, AccordionSummary, AccordionDetails, Grid } from "@mui/material";
+import { useContext, useMemo, useEffect } from "react";
+import { CollapsedContext } from "../../CollapsedContext";
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import './PathListItem.sass';
+
+export interface PathListItemProps {
+    change: any
+    showChangeType?: string
+}
+
+export const PathListItem: React.FC<PathListItemProps> = ({ change, showChangeType = "" }) => {
+    const changeVal = change.Value
```

Comment:
will you always have the value key on the change object on the first render?

Reply:
should be part of the data structure.
Why would it change?

Reply:
I don't say it would change, just want to make sure you will always have this prop and the site won't crash.
In this case, we can use change?.Value